src/main/java/seedu/contax/logic/commands/BatchCommand.java

```diff
@@ -91,7 +91,13 @@ public CommandResult execute(Model model) throws CommandException {
                         commandInput, Integer.toString(index.getOneBased()));
                 logger.info("----------------[BATCH COMMAND][" + commandText + "]");
                 Command command = addressBookParser.parseCommand(commandText);
-                commandResultList.add(command.execute(model));
+                try {
+                    commandResultList.add(command.execute(model));
+                } catch (CommandException ce) {
```

Comment:
You need to handle revert also for batch.

Reply:
the revert for range is actually simulate the first occurence at top instead of bottom as revers order, batch is normally executed shd no need

Reply:
both range and batch cahgne to revert